src/utils/generate_structs.jl

```diff
@@ -35,21 +35,14 @@ mutable struct {{struct_name}}{{#parametric}}{T <: {{parametric}}}{{/parametric}
     end
     {{/inner_constructor_check}}
 end
-
```

Comment:
Did you mean to remove these blank lines?  Now the functions in the generated code have no whitespace between them.